gno.land/cmd/gnoland/secrets_verify_test.go

```diff
@@ -184,24 +126,24 @@ func TestSecrets_Verify_All_Missing(t *testing.T) {
 			initArgs := []string{
 				"secrets",
 				"init",
-				"--data-dir",
-				tempDir,
+				"--home",
```

Comment:
If we use a single flag, I think we should use `—data`

Reply:
I believe it could lead to confusions, because the folder contains datas, configurations, secrets...